### DISCUSSION TITLE
Chat - Priority.

### DISCUSSION THREAD

**Discussion by Aides123** [Zrips/CMI#7721] [Feature Requests]
Hey!
I have a small problem with the CHAT module.

I'm using "FunnyGuilds" plugin which adds private chat for team, allies. However, the problem is that when a player tries to use this chat via the !text shortcut he gets sent to the global and guild chat. Is it possible to add weight/priority for CMI chat?

Message with !1231 she should not appear, but the one after her
![image](https://user-images.githubusercontent.com/23367220/143606739-d76298bc-eed9-4183-ac91-332a82dfdc2a.png)

**Comment by CloudeLecaw** [comment]
Pls move it to discussions :)
